ArjunaJTA/narayana-jta-jakarta/pom.xml

```diff
@@ -1,136 +1,161 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do you think that we can take advantage of this PR to add the new copyright in this file?

Reply:
The update to this file was by mistake. I searched for all poms in directories with the `-jakarta` suffix and edited them all forgetting that this pom was contributed by someone else in an earlier commit.

Reply:
In the new PR I removed the edits to this file.